android/src/main/java/io/sentry/capacitor/SentryCapacitor.java

```diff
@@ -132,7 +150,37 @@ public void startWithOptions(final PluginCall capOptions) {
             }
         );
 
-        capOptions.resolve();
+        JSObject resp = new JSObject();
+        resp.put("value", true);
+        capOptions.resolve(resp);
```

Comment:
What is this `"value"` here for? 🤔

Reply:
Capacitor only allows promises to return their own JSObject type instead of a string, so I unfortunately can't resolve with just "true".